ims-examples/epub-sr-local.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <script src='http://localhost:5000/js/deps/require.js' data-main='http://localhost:5000/js/profile-ims-default' async class='remove'></script>
```

Comment:
Should this be referencing localhost?

Reply:
After seeing prior notes about the `-local` indicator, I understand the issue now.